mmdet/models/builder.py

```diff
@@ -1,67 +1,47 @@
 import warnings
 
-from mmcv.utils import Registry, build_from_cfg
-from torch import nn
-
-BACKBONES = Registry('backbone')
-NECKS = Registry('neck')
-ROI_EXTRACTORS = Registry('roi_extractor')
-SHARED_HEADS = Registry('shared_head')
-HEADS = Registry('head')
-LOSSES = Registry('loss')
-DETECTORS = Registry('detector')
-
-
-def build(cfg, registry, default_args=None):
-    """Build a module.
-
-    Args:
-        cfg (dict, list[dict]): The config of modules, is is either a dict
-            or a list of configs.
-        registry (:obj:`Registry`): A registry the module belongs to.
-        default_args (dict, optional): Default arguments to build the module.
-            Defaults to None.
-
-    Returns:
-        nn.Module: A built nn module.
-    """
-    if isinstance(cfg, list):
-        modules = [
-            build_from_cfg(cfg_, registry, default_args) for cfg_ in cfg
-        ]
-        return nn.Sequential(*modules)
-    else:
-        return build_from_cfg(cfg, registry, default_args)
+from mmcv.cnn import MODELS as MMCV_MODELS
```

Comment:
Due to parameter initialization refactored, MMCV needs to modify `build_model_from_cfg`, E.g modify `nn.Sequential(*modules)` to `from ..runner import Sequential ; Sequential(*modules)`.

Reply:
Should create a PR fast in MMCV for 1.3.2